src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/OpprettTestBehandlingController.kt

```diff
@@ -115,6 +120,18 @@ class OpprettTestBehandlingController(
         barnService.opprettBarn(behandlingBarn)
     }
 
+    private fun opprettOppgave(behandling: Behandling) {
+        taskService.save(
+            OpprettOppgaveForOpprettetBehandlingTask.opprettTask(
+                OpprettOppgaveForOpprettetBehandlingTask.OpprettOppgaveTaskData(
+                    behandlingId = behandling.id,
+                    saksbehandler = SikkerhetContext.hentSaksbehandler(),
+                    beskrivelse = "Behandle behandling i ny løsning",
```

Comment:
Kan vi kanskje skrive at det er er testbehandling (ikke lagd med en ekte søknad)? Ser for meg at det kan være fint å se når vi begynner å kunne sende inn søknader.